op-node/rollup/types.go

```diff
@@ -307,10 +333,14 @@ func (c *Config) Description(l2Chains map[string]string) string {
 	if l2Chains != nil {
 		networkL2 = l2Chains[c.L2ChainID.String()]
 	}
+	// replace using opBNB networks
+	networkL2 = NetworkNames[c.L2ChainID.String()]
 	if networkL2 == "" {
 		networkL2 = "unknown L2"
 	}
 	networkL1 := params.NetworkNames[c.L1ChainID.String()]
+	// replace using bsc networks
+	networkL1 = NetworkNames[c.L1ChainID.String()]
```

Comment:
Here lint complains about `ineffectual assignment to networkL1 (ineffassign)`
If you want to replace bsc's networks, I don't think line 341 is necessary anymore.